docker/Dockerfile

```diff
@@ -81,6 +83,18 @@ RUN git clone https://github.com/pantoniou/libfyaml.git
 RUN cd libfyaml && ./bootstrap.sh && ./configure --prefix /usr && make && make install
 RUN ldconfig
 
+# OSTREE with libcurl support for HSM
+#  This allows ostree calls to libcurl to be able to keys controlled by pkcs11
+WORKDIR /tmp
+RUN mkdir ostree && cd ostree && git init && git remote add origin https://github.com/ldts/ostree && \
```

Comment:
Maybe it makes sense to pull `ostree` from `https://github.com/ldts/ostree` and use the version we currently use in LmP, which is `v2020.7`. Your version/fork might be useful for debugging, I suppose. If you prefer to use it then maybe it makes sense to push/fork it to github.com/foundriesio/ostree?

Reply:
argh, that shouldnt be there. my fault. will fix.